packages/base-styles/_z-index.scss

```
@@ -33,8 +33,8 @@ $z-layers: (
 	".interface-interface-skeleton__content": 20,
 	".edit-widgets-header": 30,
 	".block-library-button__inline-link .block-editor-url-input__suggestions": 6, // URL suggestions for button block above sibling inserter
-	".wp-block-cover__inner-container": 32, // InnerBlocks area inside cover image block. Must be higher than block popover and less than drop zone.
-	".wp-block-cover.is-placeholder .components-placeholder.is-large": 32, // Cover block resizer component inside a large placeholder.
+	".wp-block-cover__inner-container": 1, // InnerBlocks area inside cover image block. Must be higher than block popover and less than drop zone.
```

Nit: now that it's lowered, I think the second sentence is no longer needed?

```suggestion
	".wp-block-cover__inner-container": 1, // InnerBlocks area inside cover image block.
```

---

🤦